app/helpers/proxmox_vm_cloudinit_helper.rb

```diff
@@ -41,4 +41,49 @@ def parse_server_cloudinit(args)
     end
     cloudinit_h
   end
+
+  def create_cloudinit_iso(vm_name, configs)
+    iso = File.join('/var/lib/vz/template/iso', "#{vm_name}_cloudinit.iso")
+    arguments = ["-output #{iso}", '-volid', 'cidata', '-joliet', '-rock']
+    ssh = Fog::SSH.new(URI.parse(fog_credentials[:proxmox_url]).host, fog_credentials[:proxmox_username].split('@')[0], { password: fog_credentials[:proxmox_password] })
+    wd = ssh.run([%({mktemp -d})]).first.stdout.chomp
```

Comment:
if 'wd' is empty what happens then? 
will it later on run "rm -fr" for nil?

Reply:
mktemp -d creates a directory in tmp directory with random name. Is it possible that it can be empty?

Reply:
if this request fails because of empty space / no more resources / access rights?

Reply:
Ok, I can add exception if this happens and a check that `wd` is not null.

Reply:
Fixed